bases/br_capes_bolsas/mobilidade_internacional/table_config.yaml

```diff
@@ -0,0 +1,984 @@
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id: br_bd_diretorios_mundo
+          table_id: pais
+          column_name: nome
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: sigla_moeda
+      bigquery_type: string
+      description: Sigla da Moeda de Pagamento
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: area_avaliacao
+      bigquery_type: string
+      description: Área de avaliação
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: area_conhecimento
+      bigquery_type: string
+      description: Área do conhecimento
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: grande_area_conhecimento
+      bigquery_type: string
+      description: Grande área do conhecimento
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: nivel_ensino
+      bigquery_type: string
+      description: Nível de Ensino
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations: 'Exemplo: Graduação, doutorado, pós-dourado, etc'
+      is_in_staging: true
+      is_partition: false
+    - name: uf_instituicao_origem
+      bigquery_type: string
+      description: Unidade da Federação da instituição de origem
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id: 
+          table_id: 
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: instituicao_ensino_origem
+      bigquery_type: string
+      description: Nome da instituição de origem
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
```

Comment:
Sugestão: inserir o diretório `basedosdados.br_bd_diretorios_brasil.instituicao_ensino_superior` (se for instituição de ensino superior). Se for escola, `basedosdados.br_bd_diretorios_brasil.escola`. Fiquei na dúvida sobre isso, talvez deixar mais claro na descrição.

Reply:
Acho que todos são superior, mas não tenho certeza.